examples/spark/README.md

```diff
@@ -97,7 +97,7 @@ $ kubectl create -f examples/spark/spark-worker-controller.json
 
 ```shell
 $ kubectl get pods
-NAME                                           READY     REASON    RESTARTS   AGE
+NAME                                            READY     REASON    RESTARTS   AGE
```

Comment:
Thanks. I have another PR that changes all "REASON" to "STATUS". I expect to see a merge conflicts here :)